secure_tunneling/tests/SecureTunnelTest.cpp

```diff
@@ -78,7 +79,7 @@ static int before(struct aws_allocator *allocator, void *ctx)
     testContext->secureTunnel = new Aws::Iotsecuretunneling::SecureTunnel(
         allocator,
         nullptr,
-        nullptr,
+        s_socketOptions,
```

Comment:
If you're not going to configure this, you could just pass in `Aws::Crt::Iot::SocketOptions()`

Reply:
Ok. Will do.